components/brave_ads/browser/bundle_state_database.cc

```diff
@@ -635,11 +635,7 @@ bool BundleStateDatabase::MigrateV2toV3() {
 
 bool BundleStateDatabase::MigrateV3toV4() {
   std::string sql = "ALTER TABLE ad_info ADD advertiser_id LONGVARCHAR;";
-  if (!GetDB().Execute(sql.c_str())) {
-    return false;
-  }
-
-  return false;
+  return GetDB().Execute(sql.c_str());
```

Comment:
Great work, thank you. For this PR could you also resolve `BundleStateDatabase::MigrateV1toV2` as the final `return false;` should also be `return true;`

Reply:
Ah I guess I stated at a later migration in the first place to not hit that one. Perhaps that's related to the other crash reports I see in the tracker.